Code/GraphMol/molopstest.cpp

```diff
@@ -4643,6 +4643,7 @@ void _renumberTest(const ROMol *m) {
     }
 
     std::string nSmi = MolToSmiles(*nm, true);
+    if (nSmi != refSmi) std::cerr << refSmi << std::endl << nSmi << std::endl;
```

Comment:
Yeah, it's one of those "if it fails you're going to want to see this anyway, so might as well go ahead and show it"

This pattern is frequent enough that you'd think I'd add something like TEST_EQUAL() or switch to an actual testing framework. ;-)